src/Motor.Extensions.Hosting.RabbitMQ/RabbitMQMessageConsumer.cs

```diff
@@ -18,20 +18,21 @@ public class RabbitMQMessageConsumer<T> : IMessageConsumer<T> where T : notnull
         private readonly IHostApplicationLifetime _applicationLifetime;
         private readonly IApplicationNameService _applicationNameService;
         private readonly RabbitMQConsumerOptions<T> _options;
-        private readonly IRabbitMQConnectionFactory<T> _connectionFactory;
         private readonly ILogger<RabbitMQMessageConsumer<T>> _logger;
         private bool _started;
         private IModel? _channel;
         private CancellationToken _stoppingToken;
 
+        public IRabbitMQConnectionFactory<T> ConnectionFactory { get; }
```

Comment:
nit: make this property `internal` and allow internals to be visible to the tests to avoid the usage of this property outside of Motor.NET?